src/query.rs

```diff
@@ -226,4 +226,48 @@ impl Query {
             Err(e) => Err(to_pyerr(e)),
         }
     }
+
+    #[staticmethod]
+    #[pyo3(signature = (doc_address, min_doc_frequency = Some(5), max_doc_frequency = None, min_term_frequency = Some(2), max_query_terms = Some(25), min_word_length = None, max_word_length = None, boost_factor = Some(1.0), stop_words = vec![]))]
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I would suppress this warning, but do you have any thoughts?

Reply:
I'm ok with this.